### PR TITLE
Feature/int adap medication report

### PR DESCRIPTION
**Pull request by draganabogdanovic**
Fixed problems with adding report to the database.

### CODE REVIEW

File: Care-CureHospital/Backend/Model/DoctorMenager/Report.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    {
+        public int Id { get; set; }
+        
+        public String MedicationName { get; set; }
+
+        public int Quantity { get; set; }
+
+        public DateTime Date { get; set;  }
+
+        public Report(int id, string name, int quantity)
```
Comment (JovanBosnic7) [inline Care-CureHospital/Backend/Model/DoctorMenager/Report.cs:19]: You should not put keyword 'this' in constructor if name of constructor attribute is different than attribute of class itself.

File: Care-CureHospital/Backend/Model/DoctorMenager/Report.cs [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    public class Report : IIdentifiable<int>
+    {
+        public int Id { get; set; }
+        
+        public String MedicationName { get; set; }
+
+        public int Quantity { get; set; }
+
+        public DateTime Date { get; set;  }
+
```
Comment (JovanBosnic7) [inline Care-CureHospital/Backend/Model/DoctorMenager/Report.cs:18]: You should not put keyword 'this' in constructor if name of constructor attribute is different than attribute of class itself.

File: Care-CureHospital/Backend/Model/DoctorMenager/Report.cs [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        public Report(int id, string name, int quantity)
+        {
+            this.Id = id;
+
```
Comment (JovanBosnic7) [inline Care-CureHospital/Backend/Model/DoctorMenager/Report.cs:22]: You should not put keyword 'this' in constructor if name of constructor attribute is different than attribute of class itself. Line spacing should be smaller.

File: Care-CureHospital/Backend/Repository/DirectorRepository/ReportRepository.cs [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+using Backend.Repository.MySQL;
+using Backend.Repository.MySQL.Stream;
+using Backend.Repository.DirectorRepository;
+using Repository.IDSequencer;
+using Repository;
+
+
+namespace Backend.Repository.DirectorRepository
+{
+    public class ReportRepository : MySQLRepository<Report, int>, IReportRepository
```
Comment (JovanBosnic7) [inline Care-CureHospital/Backend/Repository/DirectorRepository/ReportRepository.cs:15]: Class should not be empty, and methods should be implemented!

File: Care-CureHospital/Backend/Service/DirectorService/ReportService.cs [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+using System.Threading.Tasks;
+using Backend.Model.DoctorMenager;
+using Model.DoctorMenager;
+using Backend.Repository.DirectorRepository;
+using Service;
+
+namespace Backend.Service.DirectorService
+{
+    public class ReportService : IService<Report, int>
+    {
```
Comment (JovanBosnic7) [inline Care-CureHospital/Backend/Service/DirectorService/ReportService.cs:13]: Public fields should start with first capital letter.

**Review by JovanBosnic7 — CHANGES_REQUESTED** [review]
Correct spacing and naming.

**Review by coa98 — APPROVED** [review]

**Review by stiven98 — APPROVED** [review]

**Pull request merged**